go-controller/pkg/node/gateway_localnet_linux_test.go

```diff
@@ -285,10 +283,8 @@ var _ = Describe("Node Operations", func() {
 		app.Name = "test"
 		app.Flags = config.Flags
 		fExec = ovntest.NewFakeExec()
-		fakeOvnNode = NewFakeOVNNode(fExec)
-		fakeOvnNode.fakeExec.AddFakeCmd(&ovntest.ExpectedCmd{
```

Comment:
why this command is removed now ? is this specific to fake node ?
Also i would like understand why we no longer depend on using watch factory exists with fakeOvnNode.

Reply:
Yes, it is specific to fake node.I don't know what you mean with `using watch factory exists with fakeOvnNode`?

Reply:
Within `fakeOVN.Start`, the default network controller `PreStart` was failing. With lint we are force to check for this error. Rather than ignoring the errors, it's clear we don't need the default network controller for anything as the test still passes. So instead of using fakeOVN and everything that comes with, use just what we need. We are using the factory, but it is setup in the test rather than using fakeOVN.

Reply:
ok @jcaamano , that explains. thanks.